server/integration_tests/test_data/e2e_answer_places/californiacountieswiththehighestasthmalevels/chart_config.json

```diff
@@ -125,13 +125,49 @@
             ],
             "denom": "Count_Person",
             "title": "Deaths From Diseases of the Respiratory System in Counties of California"
+          },
+          {
+            "columns": [
+              {
+                "tiles": [
+                  {
+                    "rankingTileSpec": {
+                      "rankingCount": 10,
+                      "showHighest": true
+                    },
+                    "statVarKey": [
+                      "Count_Death_DiseasesOfTheRespiratorySystem_Male"
+                    ],
```

Comment:
respiratory is close to asthma, but why only male?

Reply:
do we return the top 8 stat vars, this is the make up stat var..